<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QDialog" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>853</width>
    <height>515</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>853</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>831</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="all_books">
    <attribute name="title">
     <string>馆藏目录</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>书名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bookname_le">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="select_btn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>文学</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>编程</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>公务员考试</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>80</y>
       <width>24</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTableView" name="select_contents_tv">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>701</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="stu_back_btn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu&quot;;
font: 13pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="record">
    <attribute name="title">
     <string>借阅记录</string>
    </attribute>
    <widget class="QTableView" name="select_record_tv">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>700</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_btn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>30</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>刷   新</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="persenal_info">
    <attribute name="title">
     <string>个人信息</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stuid_le">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stuname_le">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学院：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stu_dep_le">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>专业：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stu_spe_le">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>160</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stu_sex">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>250</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stu_passwd">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>250</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="stu_revise_btn">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>350</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>修   改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stu_update_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>350</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>确  定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stu_cancel_btn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>350</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>取   消</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
